### DISCUSSION TITLE
[Feature]: Support for Nikon NEF raw format

### DISCUSSION THREAD

**Discussion by stevenwdhu** [immich-app/immich#2347] [Feature Request]
### The feature

Would love to see support for Nikon NEF raw files. Thanks.

[NEF Sample (from raw.pixls.us)](https://raw.pixls.us/getfile.php/2222/nice/Nikon%20-%20D4S%20-%2012bit%20uncompressed%20(3:2).NEF)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
NEF is already supported.

**Reply by stevenwdhu** [reply]
It seems that I cannot do batch upload of NEF files via CLI. It prints "No local assets found".

**Reply by bo0tzz** [reply]
You might be running into this issue https://github.com/immich-app/CLI/issues/47. As a workaround for now, uploading your NEF files via the web UI should work.

**Comment by beta4better** [comment]
I upload the NEF photo token by Nikon D3, when I browse it via the browser, the photo shows quite blurry not that clear as the JPG version. 
Anyone else met the same issue?

**Reply by tonya11en** [reply]
I've got the same issue.

**Reply by akropp** [reply]
Same issue.  Seems the NEF processing isn't very good. (at least for Nikon)

**Comment by ishridharhegde** [comment]
Running into the same issue

Unable to view .NEF images:

<img width="1455" height="725" alt="image" src="https://github.com/user-attachments/assets/44a01dc4-6859-4b5a-a72f-350812800160" />

**Reply by DungGramer** [reply]
> Running into the same issue
> 
> Unable to view .NEF images:
> 
> <img alt="image" width="1455" height="725" src="https://private-user-images.githubusercontent.com/37546564/518976395-44a01dc4-6859-4b5a-a72f-350812800160.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzI0NzA5NTYsIm5iZiI6MTc3MjQ3MDY1NiwicGF0aCI6Ii8zNzU0NjU2NC81MTg5NzYzOTUtNDRhMDFkYzQtNjg1OS00YjVhLWE3MmYtMzUwODEyODAwMTYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAzMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMzAyVDE2NTczNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdiOTQzMmE4ZTQzYmVhMDEzN2I5MjEyY2EzZjQwNjU2YTNiNTU0MzYxOTZjY2FkOWU5NmMzMmRiMmU5MmY1ZjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tsHWm4RjuSQ22PteHJWe-l3buxrXl8dwvoVM5VR67p0">

1. Goto Administration -> Image Settings -> Check to "__Prefer embedded preview__".
<img width="1058" height="664" alt="image" src="https://github.com/user-attachments/assets/48b247a4-5bd6-459c-a735-f8b5f08cb759" />

2. Then click to "__Refresh thumbnails__" error images.
<img width="611" height="536" alt="image" src="https://github.com/user-attachments/assets/ec17388a-6e42-4d53-a1e4-badf0efd3e30" />
